interface/main/calendar/add_edit_event.php

```diff
@@ -1010,7 +1016,7 @@ function setEventDate($start_date, $recurrence)
     $startampm = '1';
     if ($starttimeh >= 12) { // p.m. starts at noon and not 12:01
         $startampm = '2';
-        if ($starttimeh > 12) {
+        if ($starttimeh > 12 && $globle_timeformat == 1) {
```

Comment:
Wouldn't it be simpler to just use $GLOBALS['time_display_format'] here and drop the code block at the top?

Reply:
Recommend using the $GLOBALS['time_display_format'] directly.